core_types/src/dna/zome.rs

```diff
@@ -108,6 +118,21 @@ impl Zome {
             entry_types: entry_types.to_owned(),
             capabilities: capabilities.to_owned(),
             code: code.clone(),
+            bridges: None,
+        }
+    }
+
+    pub fn get_required_bridges(&self) -> Vec<Bridge> {
+        match self.bridges {
+            None => Vec::new(),
+            Some(ref bridges) => bridges
+                .iter()
+                .filter(|bridge| match bridge {
+                    Bridge::Address(b) => b.presence == BridgePresence::Required,
+                    Bridge::Trait(b) => b.presence == BridgePresence::Required,
+                })
+                .cloned()
+                .collect(),
```

Comment:
With the changes I requested, you can remove the match statement. Since iterators in Rust are lazy, an empty iterator will instantly return an empty `Vec<T>`.